feature/taildrop/integration_test.go

```diff
@@ -26,6 +26,21 @@ import (
 // TODO(bradfitz): add test between different users with the peercap to permit that?
 
 func TestTaildropIntegration(t *testing.T) {
+	t.Skip("known failing test; see https://github.com/tailscale/tailscale/issues/15970")
```

Comment:
@nickkhyl, we can merge this test first and then your PR fix can remove this t.Skip